### ISSUE TITLE
USA Aurora has a strange distribution of EXP

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#717]
USA Aurora has a strange distribution of EXP. Does this need adjusting?

```
AmericaJetAurora ExperienceValue = 200 200 400 400

AmericaJetRaptor ExperienceValue = 50 50 100 150
AmericaJetStealthFighter ExperienceValue = 100 100 200 300
AmericaVehicleComanche ExperienceValue = 50 50 100 200
```

**Comment by ExiLe-ZH** [comment]
Make vet2 300?

**Comment by commy2** [comment]
I don't think this matters much. Would avoid making changes for the sake of making changes, as that introduces unnecessary overhead to anyone that wants to make sense of the patch.

**Comment by Jundiyy** [comment]
Agree with commy, no need.

**Comment by ExiLe-ZH** [comment]
I think it should be lowered across the board, these numbers are quite ridiculous actually. If it's also the Alpha then that should stay the same, but vanilla aurora should be lowered since these aren't that great anyway and lowering given xp is what would make them a bit more attractive and less risky to use.

I think 150 200 250 300 is better for v aurora. Or even 100 150 200 250.

**Comment by xezon** [comment]
I agree, regular Aurora is not as great as Alpha Aurora and therefore would also not need to reward as much on kill.
